command.md

```diff
@@ -0,0 +1,45 @@
+# 11 linux command PP use most
+
+### 1. ls
+`ls` + [directory name] list all directories and files in current directory
+`ls -a` list all directories and files including the hidden ones
+`ls -l` besides name, list more details
+
+### 2. cd
+`cd` change directory + [/directory name]
+`cd ..` change to directory nexxt level up
+`cd /` change to root directory
+`cd ~` change to main directory of the current user
+`cd -` change to directory visited last time
+
+### 3. pwd
+`pwd` print current directory path
+
+### 4. mkdir
+`mkdir` create a directory
+
+### 5. touch
+`touch` create a file
+
+### 6. cp
+`cp` copy a file to destination directory
+`cp -r` copy a directory and everything in it to destination directory
+
+### 7. mv
+`mv` mv + [file name 1] + [file name 2] is used to rename file name 
+     mv + [file name] + [directory name] is used to move file
+
+### 8. rm
+`rm` remove files
+`rm -r` recursive remove files
```

Comment:
`rm -rf` is used more often than `rm -r`, but use it with cautious

Reply:
+1